solidity/src/Hyperlane.sol

```diff
@@ -87,18 +82,20 @@ contract Hyperlane is IHyperlane {
             index += 32;
             hyMsg.signatures[i].s = encodedHyMsg.toBytes32(index);
             index += 32;
-            hyMsg.signatures[i].v = encodedHyMsg.toUint8(index) + 27;
+            hyMsg.signatures[i].v = encodedHyMsg.toUint8(index);
```

Comment:
Usually v is not expressed as y-parity(bool), but as signature scheme(uint8). if y-parity is false, the equivalent signature scheme is often 27 and if the y-parity is true, it's 28. In our case, since it's already displayed as signature scheme, no need to add 27.